qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +367,73 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def get_particle_number_elements(mol_name, hf_data, n_active_electrons=None, n_active_orbitals=None):
```

Comment:
I still feel like these function names are too long and the use of `elements` instead of `table` is confusing. Worth thinking carefully about the best option

Reply:
UU^-1